docs/modules/mechanics/actions-triggers.mdx

```diff
@@ -39,6 +39,7 @@ In the future, some features that are currently used in Kits may be transferred
   | `id` | Unique identifier used to reference this action from other places in the XML. | <span className="badge badge--primary">String</span> |
   | `scope` | Sets the scope target an action should operate against. | `player`, `team`, or `match` |
   | `filter` | A filter that is tested before running actions inside. | [Filter](/docs/modules/mechanics/filters) |
+  | `untrigger-filter` | Filters if an untrigger event (e.g. a trigger filter stops matching) should be passed down to children, similar to [kit lending](/docs/modules/gear/kits#dynamic-kits). | [Filter](/docs/modules/mechanics/filters) | `never` |
```

Comment:
I think this description makes it sound like a totally alien thing. Maybe streamline it with the above filter, so while filter is:
`A filter that is tested before running actions inside`
this should be:
` A filter that is tested before untriggering the actions inside`

then follow-up with the note about actions untriggering similar to kit lending